src/environmentd/src/http/sql.rs

```diff
@@ -14,16 +14,37 @@ use serde::Deserialize;
 
 use crate::http::AuthedClient;
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Debug)]
+pub struct SqlRequestParam {
+    value: String,
+}
+
+#[derive(Deserialize, Debug)]
 pub struct SqlRequest {
-    sql: String,
+    query: String,
+    params: Option<Vec<SqlRequestParam>>,
```

Comment:
Since the new protocol no longer requires or supports parameter types, I'd be inclined just to make this an `Option<Vec<String>`? Or, ideally, an `Option<Vec<Option<String>>`, to handle `NULL` parameters a bit more precisely, as mentioned above.

Reply:
Good call